CMakeLists.txt

```diff
@@ -121,7 +121,7 @@ option_with_print(ENABLE_AUTO_BLAS "Enables CMake to auto-detect BLAS" ON)
 option_with_print(ENABLE_AUTO_LAPACK "Enables CMake to auto-detect LAPACK" ON)
 option_with_print(ENABLE_PLUGIN_TESTING "Test the plugin templates build and run" OFF)
 option_with_flags(ENABLE_XHOST "Enables processor-specific optimization" ON
-                  "-xHost" "-march=native")
+                  "-xHost" "-march=native" "/arch:AVX2" "/arch:AVX")
```

Comment:
CMake+Windows_compiler are smart enough to reject AVX2 flag via [`CHECK_CXX_COMPILER_FLAG`](https://github.com/psi4/psi4/blob/master/cmake/psi4OptionsTools.cmake#L103) if AVX2 instructions not valid? neat!

Reply:
Oops, I'm a bit too naive! It is just a compilation/linking check, not actual running. So a more robust solution is needed.